### PR TITLE
feat(CI): migrate from Appveyor to Travis for Windows CI

### LINKED ISSUE #397: Migrate from AppVeyor to Travis
https://blog.travis-ci.com/2018-10-11-windows-early-release

### PR DESCRIPTION
**Pull request by shriram2301**
resolves #397

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,19 +3,21 @@ dist: trusty
...
-- 8
-cache:
-  yarn: true
-  directories:
-  - node_modules
-  - app/node_modules
-  - "$HOME/.electron"
-  - "$HOME/.cache"
+- 8 
+# cache:
```
Comment (joshuef) [inline .travis.yml]: @shriram2301 does it work with cache enabled?

(If so would be nice to reenable this)

File: .travis.yml [hunk 2]
```diff
@@ -87,6 +91,8 @@ after_failure:
 # check resolution...
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then  system_profiler SPDisplaysDataType | grep Resolution
   fi
+- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then where python;
```
Comment (joshuef) [inline .travis.yml]: @shriram2301 we dont need this now (was there to debug `python` install)

File: .travis.yml [hunk 3]
```diff
@@ -95,13 +101,15 @@ before_deploy:
...
+- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then certutil -hashfile  release/*.zip SHA256 >> release/$RELEASE.txt; fi
 - export SHA_256_SUM="$(ls release/*.txt)";
 
 
 deploy:
   provider: releases
   skip_cleanup: true
-  api_key: $GITHUB_ACCESS_TOKEN
+   api_key:
+    secure: koxiDMX+Hl5fNd5TliLzxKI0H0eIs0elJVs686cfFNiyqnOsBS6wimxC2YK62wk/eXaHZ7gvi1Bfy6GuRnP2fNeLTL3IZ9O9HOcUP2X/jQ12fn5wkewr6DjQx3xlN+/BGSq9IAROMq5ol0moKFBFsu4rX28ovIl2Ft6TIJio8SflM7zZFI4WJa8NNGRBkkr0Dz+MB7fuMWrI3w0paj7BeHoz5ETsmW122plpAgaMQvmpB5k+ZhpCQYE0BwEEdXPAjYLOd6MQl99b0A+bg5ZC4/aJHC6UnCMdtgnzIGSrsNOdLCR8tz6tGv4rlf2TnxH9hKTKqz1N2GmQ4MSBukQpikb6DLcQbbOhjVSPwdWWU8y/O840Y68styYdTddrXJXRnr8sBtykxje1SeB3NaTsaIlSeJu8BfTgZXjJwI46kb3kSQsKHVTDdKfn3MeXEOQpxlZ4EHYEX/ZGc3Wk/uBeGpBMqiN0svdXxZTRaxl43jiJquGsI/3Kp5Q5XPlDp/cVPpCZJjQgw8qeS3fn14axnk373IJF9biSEVdfmfml5UjGX0MwMCip0OgFWvoGlo9VqMnnLYG2ltOI85XdMnZV2DuDPSUw7ODanYhnJFTr4035+TjxpXruO7wg6cuH6YPEUJR+I7pPeSUEX40RNvYsSKW07ZTLBLCkYArTcgRSiQ8=
```
Comment (joshuef) [inline .travis.yml]: Does this then need to be updated before merge, to be a maidsafe repo specific key?

File: .travis.yml (hunk 1 above)
Comment (shriram2301) [reply .travis.yml]: It fails if cache is enabled in windows. It happens consistently so for now let's keep it disabled.

File: .travis.yml (hunk 2 above)
Comment (shriram2301) [reply .travis.yml]: @joshuef  I will remove it and commit it.

**Pull request closed without merging**